test/cat.jl

```diff
@@ -79,7 +79,7 @@ module TestCat
     vcat(dt, null_dt)
     vcat(dt, dt)
     vcat(dt, dt, dt)
-    @test vcat(DataTable[]) == DataTable()
+    @test vcat(DataTable()) == DataTable()
```

Comment:
OK, but this line can even be removed since it's now testing the same thing as `vcat(null_dt)`. Ideally the tests above would also check the returned value, but...